.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. Ensure that your code follows our code conventions: https://github.com/gojek/feast/blob/master/docs/contributing.md#code-conventions
+2. Run unit tests and ensure that they are passing: https://github.com/gojek/feast/blob/master/docs/contributing.md#running-unit-tests
+3. If your change introduces any API changes, make sure to update the integration tests scripts here: https://github.com/gojek/feast/tree/master/tests/e2e
+4. Make sure documentation is updated for your PR!
```

Comment:
Make sure you have signed the CLA https://cla.developers.google.com/clas

Reply:
added